LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -462,6 +541,13 @@ function loadSearchPrereqs() {
                             if(t_inIndicators != undefined) {
                                 for(var i in t_inIndicators) {
                                     $('#indicators_' + t_inIndicators[i].indicatorID).prop('checked', true);
+
+                                    if (t_inIndicators[i].cols !== undefined) {
+                                        for (var j in t_inIndicators[i].cols) {
+                                            console.log('#indicators_' + t_inIndicators[i].indicatorID + '_columns_' + t_inIndicators[i].cols[j]);
```

Comment:
So it mostly looks good now, just clean up this last console log unless its necessary for users. 

Other than that, I was doing some backwards compatibility with reports created before this feature was implemented with a grid field type. It doesn't work it gives me this error:
![image](https://user-images.githubusercontent.com/17956374/133103678-44db2ddd-c379-4769-96cf-6fd25fa85e5a.png)

Just make sure its also backwards compatible by moving back to master branch and creating a report with grids and saving the link then checking out the new branch and testing that link for backwards compatibility.

I think this is good to go after that is done.